### DISCUSSION TITLE
Unit Test Gradle Failed

### DISCUSSION THREAD

**Discussion by XichengSpencer** [oppia/oppia-android#4988] [Q&A (Other)] [answered]
My code run successfully on my Android Studio, but when I submit [pr](https://github.com/oppia/oppia-android/pull/4987) and the only test failed is
![Screenshot 2023-05-17 202046](https://github.com/oppia/oppia-android/assets/74568012/2b8457cb-c8c9-49f6-b2e1-4f2408eae207)

> Process 'Gradle Test Executor 5' finished with non-zero exit value 134
  This problem might be caused by incorrect test process configuration.
  Please refer to the test execution section in the User Manual at https://docs.gradle.org/6.8/userguide/java_testing.html#sec:test_execution

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:testDebugUnitTest'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$3(ExecuteActionsTaskExecuter.java:186)
...
BUILD FAILED in 13m 40s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.8/userguide/command_line_interface.html#sec:command_line_warnings

Is it pure gradle file problem, and what can I do with it?

**Comment by adhiamboperes** [comment, marked as answer]
Hi @XichengSpencer, sometimes the CI run times out, and will need to be restarted manually, like in this instance. 

Here is the link to the Wiki on how to intepret the CI  results: https://github.com/oppia/oppia-android/wiki/Interpreting-CI-Results

**Comment by XichengSpencer** [comment]
Thanks, I didn't find any option to rerun the CI on the GitHub site. Should I wait for the reviewer to do that?
The error message is that some packages are deprecated, and I am not sure how to handle that as I didn't modify any gradle files in my pr.

**Reply by MohitGupta121** [reply]
Hey @XichengSpencer, the reviewer will rerun the failed test if it's due to timeout at time of merging. Regarding packages are deprecated in gradle packages you can ignore them. Thanks!
